os/se/ameba/rtl_se_smccc_wrapper.c

```diff
@@ -346,3 +346,25 @@ int ameba_hal_SignatureVerify(uint32_t *imagehdr_addr)
 	return ret;
 }
 
+/* Kernel Image Verification */

```

Comment:
This is a common file of ameba chips. Is this ok to add this without any chip config?

Reply:
Hi @sunghan-chang ,

This file only will build for ameba smart.
![image](https://github.com/Samsung/TizenRT/assets/94035727/86e83166-c584-4634-bd0f-d8bbb4231948)
The chio config was covered at the /os/se/ameba/Make.defs